src/components/components/ebay-notice-base/component.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+    onCreate() {
+        this.state = {
+            dismissed: false,
+        };
+    },
```

Comment:
Not sure how I feel about this. 

We used to have a `hidden` property, which when it was set to true it would hide the notice. 
I think we can go back to that, but we will need to make sure it syncs up with the user input. 
The problem is once it is dismissed, there's no way the user can show the notice again. 

So maybe we can do this:
```
    onInput(input) {
        this.state = { dismissed: input.dismissed || false };
    },
```
And then add a new input.dimissed to page notice. 

This way, the user can toggle the dismissed attribute, in case they want to reset it or not.

Reply:
This was intentional actually. I was basing my design decisions here on a true "dismiss" and those don't typically need to be re-displayed. When users dismiss something, they're usually completely getting rid of it from the screen and it's usually implied that this is going away for good. If you think it's worth creating this additional level of abstraction, I suppose we can. Just not sure about its utility.

Reply:
So I agree in theory with what you have. 
The concern I have is on app teams. I know a lot of app teams tend to reuse our components in different situations. It might not be ideal, but I would hate it for them to dismiss a component and then expect it to be rendered later. Having a way to basically show the notice that was dismissed again is helpful imo. 

@ianmcburnie any input? I'm fine to go either way.

Reply:
Oh, I see. You were thinking of implementers (devs), not actual end users. I was just doing a page refresh to get it back when building 😄 .

If it's an enhancement for devs, I guess I can go either way on this too but I still wonder how many devs would just refresh the page anyway.

Reply:
Agree with Andrew, that we've seen that quite often devs want to re-use/re-show the same notice component. I think we should allow that flexibility, otherwise we should remove the notice from the DOM to make it very clear once it's gone it's gone (and they would need to instantiate a new ebay-notice).